### PR TITLE
initial attempt at countdown timer

### PR DESCRIPTION
**Pull request by chiester**

### CODE REVIEW

File: functions/countdown-timer.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { Handler, HandlerEvent } from '@netlify/functions';
+import fetch from 'node-fetch';
+import { DateTime } from 'luxon';
+
+const months = [
```
Comment (ttypic) [inline functions/countdown-timer.ts:5]: We have some date utils already and you could use `import { month } from './utils/date-utils';` here

File: functions/verify-phone.ts [hunk 2]
```diff
@@ -5,7 +5,6 @@ import fetch from 'node-fetch';
 export const handler: Handler = async (event: HandlerEvent) => {
   const payload = JSON.parse(event.body);
   const { access_key, number, country_code, format } = payload || {};
-  console.log(number);
```
Comment (ttypic) [inline functions/verify-phone.ts:8]: Could you please merge or rebase main branch? this one already fixed

File: package.json [hunk 3]
```diff
@@ -8,6 +8,7 @@
   "homepage": "https://github.com/chatfuel-lab/chatfuel-json-api-examples",
   "devDependencies": {
     "@types/jest": "^26.0.23",
+    "@types/luxon": "^1.26.5",
```
Comment (ttypic) [inline package.json:11]: Could you add new libraries via yarn? and update `yarn.lock` file?

**Comment by chiester** [comment]
`getDateInTimezone(timezone_short, futureDate); // timezone_short is "EDT"`

`RangeError: Invalid time zone specified: EDT`

File: functions/verify-phone.ts (hunk 2 above)
Comment (chiester) [reply functions/verify-phone.ts:8]: I did merge from upstream and it was still there for me?

**Comment by ttypic** [comment]
Yes `getDateInTimezone` works only with canonical timezone name. I just suggested to use `months` array. Maybe this function doesn't work well for your task

File: functions/verify-phone.ts (hunk 2 above)
Comment (ttypic) [reply functions/verify-phone.ts:8]: yep, I've added some code in order to we can easily test PR and now you 9 commits behind 
image

Could I ask you one more thing: close this PR and create new one. Test action will create only on fresh PR

**Pull request closed without merging**